### DISCUSSION TITLE
Is it possible to configure multiple Auth providers of same type?

### DISCUSSION THREAD

**Discussion by deepakmohanakrishnan07** [concourse/concourse#7639] [Help & Support]
Dear All,

Is it possible to configure Concourse to use multiple Auth provider of same type? We have configured GHE as our Auth provider currently and inherit the RBAC model from GHE Teams model. Some of our teams in GHEC would like to use the same concourse cluster. So, we need to configure GHE and GHEC as Concourse auth providers. It would be great if you could provide any advice here?

**Comment by taylorsilva** [comment]
Sadly this is not possible. You can only configure one of each type of auth provider.

Question, GHE = Github Enterprise, right?
What does GHEC stand for then?
